tests/test_classical_mis.py

```diff
@@ -1,17 +1,20 @@
 import networkx as nx
+import pytest
 
 # Define classical solver configuration
 from mis.solver.solver import MISInstance, MISSolver
 from mis.pipeline.config import SolverConfig
 from mis.shared.types import MethodType
+from mis.pipeline.preprocessor import EmptyPreprocessor
 
 
-def test_empty_mis() -> None:
+@pytest.mark.parametrize("preprocessor", [None, EmptyPreprocessor()])
+def test_empty_mis(preprocessor: None | EmptyPreprocessor) -> None:
```

Comment:
Isn't `None` sufficient ? I mean it feels like an overkill to maintain `EmptyPreprocessor`.